src/form/v2/birth/forms/printForm/collectors.ts

```diff
@@ -0,0 +1,363 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+
+import {
+  ConditionalType,
+  field,
+  FieldConfig,
+  FieldType
+} from '@opencrvs/toolkit/events'
+import { InformantType } from '../pages/informant'
+
+export const printCertificateCollectors: FieldConfig[] = [
+  {
+    id: 'collector.requesterId',
+    type: FieldType.SELECT,
+    required: true,
+    label: {
+      defaultMessage: 'Requester',
+      description: 'This is the label for the field',
+      id: 'v2.event.birth.action.certificate.form.section.requester.label'
+    },
+    conditionals: [
+      {
+        type: ConditionalType.SHOW,
+        conditional: field('informant.relation').isEqualTo(InformantType.MOTHER)
+      }
+    ],
+    options: [
+      {
+        label: {
+          id: 'v2.event.birth.action.certificate.form.section.requester.mother.label',
+          defaultMessage: 'Print and issue to Informant (Mother)',
+          description: 'This is the label for the field'
+        },
+        value: InformantType.MOTHER
+      },
+      {
```

Comment:
If we'd like, we could save the options which are used in many selects to variables, e.g.:



```
      const fatherOption = {
        label: {
          id: 'v2.event.birth.action.certificate.form.section.requester.father.label',
          defaultMessage: 'Print and issue to father',
          description: 'This is the label for the field'
        },
        value: InformantType.FATHER
      }
```